lima-init.nix

```diff
@@ -159,7 +159,7 @@ in {
         };
 
         networking.nat.enable = true;
-    
+
```

Comment:
I removed this whitespace on `master` with f259566974cd66729708cd64d5f3f868d7cf5ac0